kotlinx-coroutines-core/jvm/test/DefaultExecutorStressTest.kt

```diff
@@ -35,4 +36,30 @@ class DefaultExecutorStressTest : TestBase() {
         }
         finish(2 + iterations * 4)
     }
+
+    @Test
+    fun workerShutsDown() = withVirtualTimeSource {
```

Comment:
By convention in this project, test function shall start with the verb `test` like `testWorkerShutsDown`.

Reply:
Yeah, sorry. I was already told that once, so I'll be sure to remember it.